_gsocproposals/proposal_TRANSFER.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Next generation of CMS data replication system
+layout: plain
+project: TRANSFER
+organization: Cornell, USA and Univ. of Bologna, Italy
```

Comment:
@hegner, done, I put CMS as organization, the univ were requested by Sergei though.